experimental/traffic-portal/src/app/shared/navigation/navigation.service.ts

```diff
@@ -167,6 +167,16 @@ export class NavigationService {
 				}
 			],
 			name: "Other"
+		}, {
```

Comment:
This is duplicating the `Configuration` entry, probably slipped through during a merge as it got moved up.

Reply:
yeah there is 2 configuration added, i removed the duplicate.

Reply:
Can you move it up to where it used to be? It was moved up since the `Configuration` entries are used more frequently  then `Other`/``Users`